### PR TITLE
Translation: refine wording in default language

### PR DESCRIPTION
**Pull request by HankAviator**
_If your pull request is a translation update, then the title of this pull must contains 'string' or 'translation'_

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -149,39 +149,39 @@
...
-    <string name="settings_summary_disable_resources">Workaround for conflicts with theming engines\n<b>WARNING:</b> Modules which try to change resources will not work when this is enabled</string>
+    <string name="settings_summary_pass_safetynet">Force add GMS and GSF to blacklist and remove from whitelist\nDisabling this option does not remove modules from blacklist\nXposed Hide is required</string>
+    <string name="settings_summary_show_modules">Show modules and manager in the app\nThis does not uncheck enabled modules</string>
+    <string name="settings_summary_stop_verbose_log">Verbose log will stop capturing until next reboot</string>
+    <string name="settings_summary_stop_log">Modules log will stop capturing until next reboot</string>
+    <string name="settings_summary_disable_modules_log">Exception stack thrown by modules will still be caught normally</string>
+    <string name="settings_summary_black_white_list">When enabled, you can apply a list in \"Xposed Hide\"\nXposed Hide(Blacklist): Selected apps will NOT load EdXposed modules\nNote: using Xposed Hide might break system-wide modules\' functions</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml]: Blacklist mode -> Xposed Hide (black)
Whitellist mode -> Xposed Hide (White)

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -149,39 +149,39 @@
...
-    <string name="settings_summary_stop_verbose_log">Verbose log will stop catch before the next reboot</string>
-    <string name="settings_summary_stop_log">Modules log will stop catch before the next reboot</string>
-    <string name="settings_summary_disable_modules_log">Exception stack thrown by the module will still be catched normally</string>
-    <string name="settings_summary_black_white_list">After opening, you can set the list in \"Xposed Hide\"\nXposed Hide(Black): Selected app NOT to load EdXposed modules\nNote: using Xposed Hide might break system-wide modules\' functions</string>
-    <string name="settings_summary_open_drawer">Open the navigation drawer automatically at startup</string>
-    <string name="settings_summary_disable_resources">Workaround for conflicts with theming engines\n<b>WARNING:</b> Modules which try to change resources will not work when this is enabled</string>
+    <string name="settings_summary_pass_safetynet">Force add GMS and GSF to blacklist and remove from whitelist\nDisabling this option does not remove modules from blacklist\nXposed Hide is required</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml:152]: ditto

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -280,22 +280,22 @@
...
     <string name="download_remove">Download and remove</string>
     <string name="module_saved">Module saved at %1$s</string>
     <string name="ignore_updates">Ignore update</string>
     <string name="ignore_updates_summ">Ignore updates for a specific module</string>
     <string name="reboot_recovery">Reboot to Recovery</string>
     <string name="reboot_bootloader">Reboot to Bootloader</string>
-    <string name="reboot_download">Reboot to Download</string>
-    <string name="reboot_edl">Reboot to Emergency</string>
+    <string name="reboot_download">Reboot to Download Mode</string>
+    <string name="reboot_edl">Reboot to EDL</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml]: This mode has many aliases, such as EDL and 9008, and E is Emergency, so I prefer to use Emergency or Emergency Mode

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -327,62 +327,62 @@
...
-    <string name="ignore_chinese_summary">Hide Chinese modules from list\nModules are still available in search window\nEnabling this option may result in slow performance</string>
+    <string name="ignore_chinese_summary">Hide Chinese modules from the list\nModules are still available in search window\nEnabling this option may result in slow performance</string>
 
     <!-- EdXp related -->
     <string name="nav_title_black_list" translatable="false">Xposed Hide</string>
     <string name="nav_title_compat_list">Compat Mode</string>
     <string name="white_list_mode_title">Xposed Hide(White)</string>
-    <string name="title_white_list">White</string>
-    <string name="title_black_list">Black</string>
+    <string name="title_white_list">Whitelist</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml:351]: ditto

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -327,62 +327,62 @@
...
+    <string name="title_black_list">Blacklist</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml:352]: ditto

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -327,62 +327,62 @@
...
     <string name="list_sort">List sorting</string>
-    <string name="notinstalled">The EdXposed framework is not installed.</string>
-    <string name="app_no_ui">This application does not provide a user interface</string>
+    <string name="notinstalled">EdXposed framework is not installed.</string>
+    <string name="app_no_ui">This app does not provide a user interface</string>
     <string name="compile_failed">Optimization failed or return value is empty</string>
     <string name="compile_failed_with_info">Optimization failed: </string>
     <string name="hook_modules">Force hook modules</string>
-    <string name="hook_modules_summary">Force add modules to white-list and remove from black-list\nClosing this option does not remove modules from white-list\nXposed Hide is required to enable</string>
+    <string name="hook_modules_summary">Force add modules to whitelist and remove from blacklist\nDisabling this option does not remove modules from blacklist\nXposed Hide is required</string>
```
Comment (MlgmXyysd) [inline app/src/main/res/values/strings.xml:385]: ditto

**Review by MlgmXyysd — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (HankAviator) [reply app/src/main/res/values/strings.xml]: Xposed Hide(Blacklist) -> Xposed Hide(Black) 

What about `black-list` and `white-list`? Do you want to keep them as-is?

Comment (MlgmXyysd) [reply app/src/main/res/values/strings.xml]: > Xposed Hide(Blacklist) -> Xposed Hide(Black)
> 
> What about `black-list` and `white-list`? Do you want to keep them as-is?

just change to `Xposed Hide (Black/White)`

**Comment by HankAviator** [comment]
Wait, forgot a commit.

**Comment by MlgmXyysd** [comment]
Changes merged in https://github.com/ElderDrivers/EdXposedManager/commit/9bcf859709c1159f7f11e9fe6df713e14596f8f4

**Pull request closed without merging**